### DISCUSSION TITLE
Can we get visual studio support

### DISCUSSION THREAD

**Discussion by Xyncgas** [FStarLang/FStar#2683] [General]
Mixing F# Fstar would be fun

**Comment by kant2002** [comment]
You can try https://github.com/kant2002/FStarMSBuildSdk this is still limiting, but maybe that allow you to get started? Also take a look at https://github.com/kant2002/fstarsample

**Comment by nikswamy** [comment]
See https://github.com/FStarLang/fstar-vscode-assistant
